lib/bcu.rb

```diff
@@ -4,11 +4,58 @@
 require "cask/all"
 require "extend/formatter"
 require "extend/cask"
+require "fileutils"
+require "set"
+
+PINS_FILE = File.expand_path(File.dirname(__FILE__) + "/../pins")
 
 module Bcu
   def self.process(args)
     parse!(args)
 
+    FileUtils.touch(PINS_FILE)
+
+    pins = Set[]
```

Comment:
Would be nice to extract this to a separate function that returns that set
```suggestion
    pins = load_pins
```